### PR TITLE
chore: add eslint

### PR DESCRIPTION
**Pull request by siqbal181**
- [x] Installed linter (`npm install eslint --save-dev`)
- [x] Added ignore certain files where it wasn't needed
- [x] Fixed existing lint issues (none involved crazy code changes)
- [x] added a ignore line for the baseUrl.js file as it was throwing a weird error that wasn't relevant

### CODE REVIEW

File: frontend/App.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import React from "react";
```
Comment (pablisch) [inline frontend/App.js:1]: Out of interest, why is this needed? Generally this has not been needed in modern react unless avoiding the direct import of things like useState.

File: frontend/src/components/Button/Button.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ import React from "react";
 import { TouchableOpacity, Text } from "react-native";
 import { styles } from "./ButtonStyles";
 
-export default Button = ({ onPress, title }) => {
+const Button = ({ onPress, title }) => {
```
Comment (pablisch) [inline frontend/src/components/Button/Button.js:5]: Interested in this change too. I am not aware of any difference. Is a linter or consistency issue?

**Review by pablisch — APPROVED** [review]
There are a couple of imports of React in these files and I am sure more in the project. We should probably agree if they are needed or not and get rid of all instances throughout if not.

File: frontend/App.js (hunk 1 above)
Comment (siqbal181) [reply frontend/App.js:1]: It told me that it was needed when using JSX

File: frontend/src/components/Button/Button.js (hunk 2 above)
Comment (siqbal181) [reply frontend/src/components/Button/Button.js:5]: linter issue but also probably helps with consistency
